### DISCUSSION TITLE
Week 3 Coding Assignment: Final Story Map

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#27] [Assignments]
Complete the in-class lab, and submit a finalized Story Map. Your Story Map should include (but not limited to) the following elements:

- A fully functional interactive map
- "Data" driven narrative, using a javascript object with key:value pairs that generate the markers, narratives, links, and images as necessary on the map
- A data-driven sidebar with clickable elements that help navigate the map-based narrative
- At least one additional feature that is not covered in the lab that makes your map "different." This could be a unique design/layout of the site, or it could be a functionality component that adds to the user experience. Be creative!

**Comment by kimjee8955** [comment]
https://kimjee8955.github.io/DH151/Week3/index.html 
Features I added to make my map different: 
- changed button (side-bar item) color on hover
- added pressed effect on buttons when clicked
- added paragraph below each button when clicked, paragraph disappears when button clicked again
- added default view button on leaflet map
- changed icon color in the map to violet

**Comment by eliaranita1** [comment]
https://eliaranita1.github.io/DH151/week%203/index.html
added:
changes sidebar item appearance on mouseover
banner image

**Comment by lisa58** [comment]
https://lisa58.github.io/DH151/Week3/index.html
Changed:
when you click on the sidebar buttons, the titles appear and it stays that way until the browser is refreshed
when you hover over the sidebar buttons, the titles appear

**Comment by justisg** [comment]
https://justisg.github.io/DH151/Week3/
New map icons were implemented

**Comment by alicelu170** [comment]
https://alicelu170.github.io/DH151/Week%203/index.html

Added lines connecting the locations in order.

**Comment by anncong** [comment]
https://anncong.github.io/DH151/Week3/Index.html 

**Added:**
- hover shadows to show users they are able to click on the block element
- Illustration from internet (attributed)

**Comment by derekwang99** [comment]
https://derekwang99.github.io/DH151/Week%203/index.html

Additions:
1. Added pictures to the markers
2. Changed background color upon hovering on sidebar item.
3. Added gif as background for header

**Comment by lisalou36** [comment]
https://lisalou36.github.io/DH151/week%203
Additions:
1. Changed font of header and sidebar and buttons
2. Changed cursor: now it has the pointer option for clickable items
3. Added highlight to sidebar buttons when cursor is hovering over
4. Added polygons of different colors
5. Added to narrative, included link out to native.ca website

**Reply by jmtanenbaum** [reply]
This is really cool! I love the indigenous borders.

**Reply by hanarama** [reply]
I agree with Julia, adding the borders really adds an additional layer of interest with the narrative.

**Comment by MaryJaneHowland** [comment]
https://maryjanehowland.github.io/DH151/Week%203/index.html

Additions:
1. Made my marker the shape of a sail boat
2. Added a link to a video of me and teammates sailing to the page

**Comment by danielliuzy** [comment]
https://danielliu524.github.io/DH151/Week3/index.html
Additional features:
Cursor changes when hovering over a clickable object
Centered title of the website in the header
Added a show/hide sidebar button, when clicked hides or shows the sidebar
Offset the flyTo latitude by +0.01 to account for the size of the popup

**Comment by alexthegreat1** [comment]
https://alexthegreat1.github.io/DH151/Week%203/index.html

Additional features:
- Added the ability to choose from different base map styles, like a light or dark mode, night, or watercolor.
- Stylized the sidebar buttons more
- Made mouse behave like a pointer when hovering over the buttons so users know they're clickable
- Added a shadow that displays when users hover over the buttons

**Comment by ivanma9** [comment]
https://ivanma9.github.io/DH151/Week3/

Additional Features:

- Image moved to hover on bottom left of page.
- Sidebar on right side
- Hover on side bar item will show the description.
- Centered title

**Comment by natgrace** [comment]
https://natgrace.github.io/DH151/Week3/index.html

-  Hovering on sidebar items changes the color of their borders
- Changed marker icons
- Added to the latitude so the popup doesn't get cut off 
- Shifted header to the same column as sidebar so the map is bigger
- Centered text + image in popups

**Comment by kaisanwatanabe** [comment]
https://kaisanwatanabe.github.io/DH151/week%203/index.html

- Changed color of buttons on hover
- Reorganized grid to change look of site
- Connected markers using lines

**Comment by jmtanenbaum** [comment]
https://jmtanenbaum.github.io/DH151/week3/index.html 
I added a shadow on hover, a gradient sidebar, images and attributions, and custom book icons from the Noun Project.

**Comment by ryanorlando** [comment]
https://ryanorlando.github.io/Dh151/Week3/index.html
Features I added:

- Custom markers
- Pictures on each popup
- Expanded sidebar slightly to better fit text

**Comment by jomh17** [comment]
https://jomh17.github.io/DH151/WEEK%203

-changed the color markers
-added a hover function to the sidebar 
-made the header and sidebar gradient
-changed the shape of the city names in the sidebar

**Comment by JuliaWood1** [comment]
https://juliawood1.github.io/DH151/Week3/index.html 

Things I added:
1. Pictures to the markers
2. Red lines connecting the places in order of how we visited them
3. Dates of when we were in each city
4. The fonts of the lettering in the header and sidebar
5.  Making the buttons appear clickable to improve UX

**Comment by adrianaromero819** [comment]
https://adrianaromero819.github.io/DH151/Week3/index.html

**Additional Features**
- Centered text
- Made cursor on the sidebar a pointer
- Changed markers to transparent stars
- The text in the sidebar-items change color when you hover over them

**Comment by hoz-map** [comment]
https://hoz-map.github.io/DH151/Week3/index.html

I added background music that hopefully gives an audience the museum ambiance.

**Comment by ainsleighdouglas** [comment]
I added images, clickable interactions and little design elements
https://ainsleighdouglas.github.io/DH-151/Week3/index.html

**Comment by hanarama** [comment]
https://hanarama.github.io/DH151/Week3/index.html

Changes: 
- Rounded buttons
- Color change on hover, with transition
- Hearts instead of map markers
- Background image on header

**Comment by LynnT2** [comment]
https://LynnT2.github.io/DH151/Week%202/index.html

Additional features:
- Bruin Paw Marker Icons
- Popups in a set position to the right of its marker to ensure full visibility of marker on click
- Photos for each marker
- Rounded corners for sidebar items

**Comment by michelle14lee** [comment]
https://michelle14lee.github.io/DH151/Week3/index.html

Updates:

- Changed marker color to green
- Created color change on hovering over sidebar button
- Added images to locations
- Added to sidebar and location descriptions

**Comment by kaicolorado** [comment]
https://kaicolorado.github.io/DH151/Week%203/index.html

- Added separate description and title for markers and buttons
- Reformatted sidebar and header
- Changed font style

**Comment by sebdills** [comment]
https://github.com/sebdills/DH151/blob/aa3409c348b37b3f6ec783e2a835fe745a6bae05/Week%203/index.html

Changed the style of the pins.

**Comment by zonaliao** [comment]
https://zonaliao.github.io/DH151/Week3/index.html

I changed the hover over the sidebar buttons, added a white shadow to it, and rounded the corners

**Comment by ec028** [comment]
https://ec028.github.io/DH151/Week3/index.html

- rounded buttons
- centered title
- added descriptions back to pop ups
- fixed/improved the color scheme of the site

**Comment by frida-barragan** [comment]
https://frida-barragan.github.io/DH151/Week2/index.html

Additional features:
- changed the marker color
- created lines to connect the markers

**Reply by frida-barragan** [reply]
https://frida-barragan.github.io/DH151/practice/index.html

I added some more features to my final story map that include:
- header animation
- a scroll bar
- landing page

**Comment by kmcinerny** [comment]
https://kmcinerny.github.io/DH151/Week3/index.html

I changed the cursor to be clickable on the markers and location titles. I also adjusted the side bar styles.

**Comment by alekhyavittalam** [comment]
https://alekhyavittalam.github.io/DH151/Week3/index.html

I made the sidebar buttons change color on hover.
I changed the icons to images.

**Comment by yjwsb233** [comment]
https://yjwsb233.github.io/DH151/Week3/index.html
I made actual buttons and customized my markers
